frontend/src/component/admin/users/UsersList/UsersList.tsx

```diff
@@ -148,6 +151,21 @@ const UsersList = () => {
                 ),
                 searchable: true,
             },
+            ...(showUserDeviceCount
+                ? [
+                      {
+                          id: 'warning',
```

Comment:
are we preparing for other warnings here?

Reply:
that's an issue with old `react-table`. maybe we need to migrate fully